build.gradle

```diff
@@ -126,14 +140,14 @@ clean {
 task configure(type: Exec) {
   environment makeEnvironment
   commandLine shell
-  args shell_flag, "${set_env} && ${cmake} -B cmake-build-release ${cmakeFlags}"
+  args shell_flag, "${set_env} && ${cmake} -B ${cmake_build_dir} ${cmakeFlags}"
   workingDir "${projectDir}"
 }
 
 task compile(type: Exec) {
   environment makeEnvironment
   commandLine shell
-  args shell_flag, "${set_env} && ${cmake} --build cmake-build-release --config Release -t install -v -j ${numCpus}"
+  args shell_flag, "${set_env} && ${cmake} --build ${cmake_build_dir} --config ${cmake_build_type} -t MlSeccomp -v -j ${numCpus}"
```

Comment:
It looks like changing `install` to `MlSeccomp` was just for debugging and should be reverted now.

Reply:
oops! 🤦